include/rocksdb/advanced_options.h

```diff
@@ -1146,6 +1146,15 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through the SetOptions() API.
   uint32_t memtable_avg_op_scan_flush_trigger = 0;
 
+  // Prepare this column family for ingesting files to the last level.
+  // This option needs to be set to true before any writes to the CF.
+  // See comment above DBOptions::allow_ingest_behind for more details about
+  // the functionality and its use cases.
+  //
+  // Default: false
+  // Dynamically changeable through the SetOptions() API.
+  bool cf_allow_ingest_behind = false;
```

Comment:
How does this new option and the old option mix? Which way overrides the other? It's not clear from the API comments